.circleci/config.yml

```diff
@@ -844,6 +873,42 @@ jobs:
             timeout 1800 bash -c 'while [[ "$(curl -n -s -o /dev/null -w ''%{http_code}'' https://api.mapbox.com/downloads/v2/mobile-maps-ios/releases/ios/$VERSION/MapboxMaps.zip)" != "200" ]]; do sleep 5; done' || false
           no_output_timeout: 30m
 
+  turf-integration:
+    <<: *base-job
+    resource_class: macos.x86.medium.gen2
+    steps:
+      - checkout
+      - configure-environment
+      - turf-report-pending:
+          turf-revision: << pipeline.parameters.turf-revision >>
+      - run:
+          name: Resolve Turf
+          command: swift package resolve && swift package resolve << parameters.turf-requirement >> turf-swift
+      # Building and testing are split into 2, with the aim that we'll be able to reuse
+      # the build product and test on multiple simulators
+      - run:
+          name: Building MapboxMaps for simulator
+          command: make build-sdk-for-testing-simulator
+          no_output_timeout: 5m
+      - run:
+          name: Testing MapboxMaps with simulator
+          command: make test-sdk-without-building-simulator
```

Comment:
So we are not reusing because it would also trigger a lot of extra jobs, right?

Reply:
We also have to call additional GitHub status checks.

Reply:
Yep